src/bashly.yml

```diff
@@ -8,7 +8,6 @@ version: 0.2.1
 
 commands:
   - name: smbios
-    alias: s
```

Comment:
I am not so sure if we should change existing option, possibly breaking someone's workflow. At least unless we have very good reason to.

Reply:
I removed it, because both smbios and smmstore should use s shortcut. We can just add smmstore without shortcut and leave the shortcut for smbios if that would be better.

Reply:
> I removed it, because both smbios and smmstore should use s shortcut.

Why should, there is no universal rule here.

Reply:
restored aliases in abc24da530dcb4e4756a398fd5b4f8719e1a1a7c